docs/output.md

```diff
@@ -119,16 +145,23 @@ given barcodes/antibodies.
 
 </details>
 
-This step uses the `pixelator single-cell collapse` command.
+### Compute connected components
 
-The `collapse` command removes duplicate reads and performs error correction.
-This is achieved using the unique pixel identifier and unique molecular identifier sequences to check for
-uniqueness, collapse and compute a read count. The command generates a QC report in JSON format.
-Errors are allowed when collapsing reads if `--algorithm` is set to `adjacency` (this is the default option).
+This step uses the `pixelator single-cell graph` command.
+The input is the edge list parquet file generated in the collapse step.
+The molecules from edge list are filtered by count (`--graph_min_count`) to form the edges of the connected components of the graph.
+When graphs are computed and identified, their ID names are added back to the edge list in a column called "component".
+
+The graph command has the option to recover components (technical multiplets) into smaller
+components using community detection to find and remove problematic edges
+(see `--multiplet_recovery`). These new component IDs are then stored in the "component" column. The information to keep track of the original and
+newly recovered components are stored in a file (components_recovered.csv).
+This file is not included in the output folder by default, but can be included by passing `--save_recovered_components`.
 
-The output format of this command is an edge list in CSV format.
+The edgelist is intermediate and by default not placed in the output folder with the final files delivered to users.
```

Comment:
```suggestion
The edge list is intermediate and by default not placed in the output folder with the final files delivered to users.
```

Reply:
inconsist spelling of this throughout this document, please settle on one spelling 🙂